File: week-03/day-03/SharpieSet/src/Sharpie.java
public class Sharpie {

  String color;
  float width;
  float inkAmount;

  public Sharpie(String color, float width) {
    this.color = color;
    this.width = width;
    inkAmount = 100;
  }

  public static void use(Sharpie sharpie){
    sharpie.inkAmount--;
  }
}

File: week-09/Closures & This/Sharpie/sharpie.js
class Sharpie {
  constructor(color, width) {
    this.color = color;
    this.width = width;
    this.inkAmount = 100;
    this.use = function(){
        this.inkAmount -= this.width;
    }
  }
}

let sharpie = new Sharpie("red", 1);
let counter = 1;

while(sharpie.inkAmount > 0){
    sharpie.use();
    console.log(`This is turn: ${counter}`);
    counter++;
}
